### PR TITLE
[release/5.0] Add Fedora 36 RID

### PR DESCRIPTION
**Pull request by omajid**
It's currently in development:

    $ podman run -it registry.fedoraproject.org/fedora:rawhide /bin/cat /etc/os-release
    NAME="Fedora Linux"
    VERSION="36 (Container Image Prerelease)"
    ID=fedora
    VERSION_ID=36
    VERSION_CODENAME=""
    PLATFORM_ID="platform:f36"
    PRETTY_NAME="Fedora Linux 36 (Container Image Prerelease)"
    ANSI_COLOR="0;38;2;60;110;180"
    LOGO=fedora-logo-icon
    CPE_NAME="cpe:/o:fedoraproject:fedora:36"
    HOME_URL="https://fedoraproject.org/"
    DOCUMENTATION_URL="https://docs.fedoraproject.org/en-US/fedora/rawhide/system-administrators-guide/"
    SUPPORT_URL="https://fedoraproject.org/wiki/Communicating_and_getting_help"
    BUG_REPORT_URL="https://bugzilla.redhat.com/"
    REDHAT_BUGZILLA_PRODUCT="Fedora"
    REDHAT_BUGZILLA_PRODUCT_VERSION=rawhide
    REDHAT_SUPPORT_PRODUCT="Fedora"
    REDHAT_SUPPORT_PRODUCT_VERSION=rawhide
    PRIVACY_POLICY_URL="https://fedoraproject.org/wiki/Legal:PrivacyPolicy"
    VARIANT="Container Image"
    VARIANT_ID=container

### CODE REVIEW

**Comment by Anipik** [comment]
can you add the packaging changes here ? updating packageIndex.json and packageVersion

**Comment by omajid** [comment]
Thanks, @Anipik! Done.

**Comment by Anipik** [comment]
cc @aik-jahoda

File: src/libraries/pkg/baseline/packageIndex.json [hunk 1]
```diff
@@ -1107,7 +1107,8 @@
         "3.0.0",
         "5.0.0",
         "5.0.1",
-        "5.0.2"
+        "5.0.2",
+        "5.0.3"
       ],
       "BaselineVersion": "5.0.2",
```
Comment (Anipik) [inline src/libraries/pkg/baseline/packageIndex.json:1113]: this needs to be updated as well

Comment (omajid) [reply src/libraries/pkg/baseline/packageIndex.json]: Good point. I rebased it on release/5.0, which has picked up this (and other) changes from https://github.com/dotnet/runtime/pull/57226

**Review by wfurt — APPROVED** [review]
LGTM

**Pull request merged**